tools/local_wandb_server.py

```diff
@@ -359,6 +359,7 @@ def _start_container(
     """
     docker_flags = [
         "--detach",
+        *["--pull", "always"],  # Always get latest image.
```

Comment:
in the old implementation we allowed devs to overwrite this, sometimes people want an older server version to test  things...

Reply:
also being able to specify the tag is useful, i see that here you only allow master, let's modify it to allow using custom tags

Reply:
lastly, `/wandb-production/images/local-testcontainer` older version are removed periodically so it would be good to add back the option to point to our backup repository